spec/models/health/qualifying_activity_spec.rb

```diff
@@ -25,7 +25,7 @@
     let!(:referral_ds) { create :referral_ds }
 
     before(:each) do
-      travel_to(Date.current - 2.years) # Enrollment durations depend on time if there is no disenrollment date
```

Comment:
Looks like the test expects this to match `Health::QualifyingActivityV2::EFFECTIVE_DATE_RANGE` but Date.current has aged out.

Note I do not have much knowledge of what's going on in the Health* space